docs/getting-started/policies/application-types.rst

```diff
@@ -555,6 +555,20 @@ see the :doc:`/using/libraries/index` guides.
   Literals in Polar are constant, and any changes made to such objects
   by calling a method will not be persisted.
 
+``nil``
```

Comment:
Should we also have a section for this in each language library page? I think someone could be looking for it in either place.

Reply:
Yeah, I think we should add it to the language library pages since we mention other type translations there.

Reply:
f676bab